controllers/output.go

```diff
@@ -0,0 +1,58 @@
+package controllers
+
+import (
+	"fmt"
+
+	konfigmanagerv1 "github.com/flanksource/konfig-manager/api/v1"
+	"github.com/flanksource/konfig-manager/pkg"
+)
+
+func (r *HierarchyConfigReconciler) createOutput(output konfigmanagerv1.Output, properties map[string]string) error {
+	if output.Type == "file" || output.Type == "File" {
+		properties = getPropertiesInFileFormat(properties, output.FileName)
```

Comment:
rather pass the config object in and then run config.GeneratePropertiesFile() as it will order and add comments into the file